src/lib/utilities.js

```diff
@@ -278,3 +278,6 @@ export function filterByEnvironment(development, beta, production) {
   }
   return development;
 }
+
+export const robotsTxt = 'User-agent: *\n Disallow: /';
```

Comment:
I also just thought, lets make this variable name even clearer. Let's make it `nothingAllowedRobotsTxt` unless you have a better idea that also communicates the idea that it's not just a normal Robots.txt